bark/tools/dat-from-wav.py

```diff
@@ -0,0 +1,28 @@
+from scipy.io import wavfile as wavfile
+import bark, os
+
+
+def main(wav):
```

Comment:
You've separated the argument parsing from the primary functionality of the module, which is great.
But I think your functions' names could more clearly denote their respective purposes.
What about `wav_to_dat` or `wav2dat` rather than `main`, and `_main` rather than `_run`?